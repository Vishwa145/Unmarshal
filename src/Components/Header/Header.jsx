import React from 'react';
import logo from "../../images/logo.png"
import SearchIcon from "../../images/SearchIcon.svg";

function Header (){
    return(
        <div className="Nav">
            <div className="Logo"><img src={logo} alt="Logo"/><h2>unmarshal</h2></div>
            <div className="SearchBar"><input type="text" placeholder="Search by Address / Txn Hash" name="email"></input><button><img height="100%" src={SearchIcon} alt=""/></button></div>
        </div>
    );
}

export default Header;
