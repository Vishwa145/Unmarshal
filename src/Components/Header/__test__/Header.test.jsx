import React from "react";
import ReactDOM from "react-dom";
import Header from "../Header";
import renderer from "react-test-renderer"

it("render successfully", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Header />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("Matches snapshot", ()=>{
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
});