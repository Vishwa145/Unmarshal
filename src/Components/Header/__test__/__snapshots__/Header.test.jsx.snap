// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div
  className="Nav"
>
  <div
    className="Logo"
  >
    <img
      alt="Logo"
      src="logo.png"
    />
    <h2>
      unmarshal
    </h2>
  </div>
  <div
    className="SearchBar"
  >
    <input
      name="email"
      placeholder="Search by Address / Txn Hash"
      type="text"
    />
    <button>
      <img
        alt=""
        height="100%"
        src="SearchIcon.svg"
      />
    </button>
  </div>
</div>
`;
